### ISSUE TITLE
Select - no information about the state visible for screen readers

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12863]
**Is this a bug, enhancement, or feature request?**
bug

**Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)**
angular: 18.2.4
fundamental: 0.53.1-rc.23

**If this is a bug, please provide steps for reproducing it; the exact components you are using;**

1. Create a select  like in the example [select](https://sap.github.io/fundamental-ngx/#/core/select#semantic).
2. Turn on VoiceOver (tested with MacOs version 14.7) and set Voice Over focus on this success select and try to read state message which is put automatically as a first  option inside select.

Currently there only stateMessage is read nad information about the state is passed only for sighted users by the color. There is no information about the state for blind users.  There should be also information about the state visible for screen readers but not visible for sighted users. Eg:

'Error state: error mesage'
'Success state: success message'
'Warning state: warning message'
'Information state: information message'

It was consulted with accessibility expert.
